site/en/blog/chrome-input-ime-deprecation/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: chrome.input.ime deprecation on ChromeOS
+description: >
+  chrome.input.ime is being deprecated and will be removed no earlier than ChromeOS 119.
+layout: layouts/blog-post.njk
+authors:
+  - samrichard
+date: 2023-08-21
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/8ayEOa5luIniBq7RxKkC.png'
+alt: ''
+tags:
+  - extensions-news
+---
+
+On July 18, we [announced](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/0ybWrEVaE-I/m/8QOeRmxrBQAJ) we were ending support for [`chrome.input.ime⁠`](/docs/extensions/reference/input_ime/) on ChromeOS as of ChromeOS 117, scheduled for September. After conversations with the developer community, we’ve pushed back the removal of `chrome.input.ime` to ChromeOS 119 at the earliest. We hope that this lengthier timeline will give you adequate time to notify your users and clients of the upcoming change.
```

Comment:
I think if we're going to post the whole post, we need to intro it to make it clear who this is for. So something like,

"This post, originally posted on the ChromeOS site, contains information for Chrome Extensions developers who use `chrome.input.ime` on ChromeOS."

(or something along those lines)

Essentially, making sure that anyone this doesn't apply to can just move right along.

Reply:
I added 'extension API' to clarify that this is not talking about anything related to the web platform. I don't think it's necessary to state that this is a reprint, especially since the only reason Sam published on the ChromeOS blog is because he didn't know he could publish it on our site.

Reply:
I disagree, as this is still a slightly unusual post for us to publish. Anyway, I ask for all posts to have a clear intro as to who they are for due to the wide range of audiences we support, so please add something along the lines that I mentioned—thanks!

Also, make sure the "we" is clear. It looks like "we" is the "ChromeOS team" in this context.